openstack-clients/windows/Source_OpenRC.ps1

```diff
@@ -38,12 +38,12 @@ Else {
     #
     # *NOTE*: Using the 2.0 *auth api* does not mean that compute api is 2.0.  We
     # will use the 1.1 *compute api*
-    $os_project_name = Select-String -Path $openrc -Pattern 'OS_PROJECT_NAME'
+    $os_project_name = Select-String -Path $openrc -Pattern 'export OS_PROJECT_NAME'
     If ($os_project_name) {
         $env:OS_PROJECT_NAME = ([string]($os_project_name)).Split("=")[1].Replace("`"","")
     }
     Else {
-        Write $error
+        Write $rcerror
```

Comment:
The variable on line 32 should also be renamed to `$rcerror`, otherwise PowerShell does not print an error message on failure.

Reply:
Updated.. thanks for the catches.  

As for write.. yes I will change this to write-host.